data/core/encyclopedia/geography.cfg

```diff
@@ -161,25 +172,37 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
     text= _ "Cold, harsh, and inaccessible, the Far North is the ancestral home of the Orcish Clannate. It lies north of the <ref>dst='heart_mountains' text='Heart Mountains'</ref>, which the Orcs call the Haggid-Dargor and claim (without merit) as their own. To the east lie the Unaligned Tribes of the Wild Steppe, who fell out of the control of the Clannate, instead roaming with wild human barbarians and clashing with the High Elves of the North Plains (known as North Elves in human lands). The High Elves themselves reside further east, where it is rumored they rule a vast kingdom.
```

Comment:
Each string is presented to the translators as a single translatable block, and so any change to it results in the whole block being marked as "needs you to work out what changed and update it".

You can instead split in into separate translatable blocks at paragraphs - note that there's no translation mark on the empty-line string. After running wmlindent on it the result looks a bit messy with the indentation, but it seems a fair trade-off.

```
    text= _ "Cold, harsh, ... a vast kingdom." + "

" +
        #po: geography of the Far North
        _ "<header>text='Notable cities:'</header>
```

For strings that are being updated, it would be good to split them into blocks - this has already been done on some strings in the editor help text, but existing strings haven't been changed as it would unnecessarily break the existing translations.

Reply:
The indent is a bit odd, but it's necessary otherwise wmlindent complains (the wmlindent tool will intent it as expected).

For strings that don't need a translation hint, the string can start on the same line as the `+` sign, however I feel the "Notable cities" sections ought to have a hint about where in the map they are.

Without a po hint, this works and keeps wmlindent happy:
```
    text= _ "The Great River forms the boundary between the <ref>dst='kingdom_wesnoth' text='Kingdom of Wesnoth'</ref> and the <ref>dst='northlands' text='Northlands'</ref>. It empties to the <ref>dst='great_ocean' text='ocean'</ref> at <ref>dst='elensefar' text='Elensefar'</ref>." + "

" + _ "The Ford of Abez is a crossing between Wesnoth and the Northlands, and is the westernmost point where the Great River can be crossed without ships."
```